jplag/src/main/java/de/jplag/JPlagComparison.java

```diff
@@ -76,7 +77,7 @@ public final int getNumberOfMatchedTokens() {
         int numberOfMatchedTokens = 0;
 
         for (Match match : matches) {
-            numberOfMatchedTokens += match.getLength();
+            numberOfMatchedTokens += match.length();
```

Comment:
Quite a few `getX()` methods are replaced by `x()`. Didn't we just make the changes from `x()` to `getX()`? (might have been one of your PRs?)

Reply:
Records :)